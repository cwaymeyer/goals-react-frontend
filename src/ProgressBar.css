#goalpage-progress-bar {
  height: 30px;
  width: 100%;
}
