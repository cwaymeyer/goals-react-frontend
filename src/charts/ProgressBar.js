import "./ProgressBar.css";
import { Grid } from "@mui/material";

function ProgressBar({ ormPercentage, lbsImproved, lbsLeft }) {
  return (
    <>
      <Grid container>
        <Grid item xs={6}>
          <h3>Progress</h3>
        </Grid>
        <Grid item xs={6}>
          <h4>
            Increased {lbsImproved} lbs &nbsp;&nbsp;|&nbsp;&nbsp; {lbsLeft} lbs
            to go
          </h4>
        </Grid>
        <progress id="goalpage-progress-bar" value={ormPercentage} max="100">
          {ormPercentage}
        </progress>
      </Grid>
    </>
  );
}

export default ProgressBar;
