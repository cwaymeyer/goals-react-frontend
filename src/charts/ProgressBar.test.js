import { render } from "@testing-library/react";
import ProgressBar from "./ProgressBar";

it("renders without crashing", function () {
  render(<ProgressBar />);
});

it("matches snapshot", function () {
  const { asFragment } = render(<ProgressBar />);
  expect(asFragment()).toMatchSnapshot();
});
