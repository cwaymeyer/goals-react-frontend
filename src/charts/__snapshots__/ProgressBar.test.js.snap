// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
    >
      <h3>
        Progress
      </h3>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 css-1osj8n2-MuiGrid-root"
    >
      <h4>
        Increased  lbs   |    lbs to go
      </h4>
    </div>
    <progress
      id="goalpage-progress-bar"
      max="100"
    />
  </div>
</DocumentFragment>
`;
